labeler/src/main.rs

```diff
@@ -0,0 +1,52 @@
+use regex::Regex;
+use std::fs;
+
+mod github;
+
+fn issue_list() -> Vec<usize> {
+    let mut issue_list = Vec::new();
+    for path in fs::read_dir("./ices").unwrap() {
```

Comment:
It'd be worth making `glacier::discover` public and using that so they're both on the same page as what counts as an ICE

Reply:
Hmm, but `discover` doesn't put numbers out. Indeed these two functions are similar but I'm not sure that we should use `discover` here.

Reply:
The issue number is within

https://github.com/rust-lang/glacier/blob/7d48de21a4573b97d29119d3fb288d69b7b3cc1a/src/lib.rs#L117-L126

You'd be able to extract that to it's own method on ICE that issue_url calls

Reply:
Uhmm, I think `fn issue_list` is simple enough so we don't need to call an external method or function.

Reply:
Sure, we can always change it if there's something added where the difference matters